src/Installers/Windows/.vsconfig

```diff
@@ -0,0 +1,17 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.Net.Component.4.6.1.TargetingPack",
+    "Microsoft.Net.Component.4.7.2.SDK",
+    "Microsoft.Net.Component.4.7.2.TargetingPack",
+    "Microsoft.VisualStudio.Component.Azure.Storage.Emulator",
+    "Microsoft.VisualStudio.Component.VC.ATL",
```

Comment:
I found the `.sln` files that contained `.vcxproj` files and added the C++ components only to them.